src/transformers/tokenization_utils_base.py

```diff
@@ -2456,8 +2456,10 @@ def save_pretrained(
 
         # Add tokenizer class to the tokenizer config to be able to reload it with from_pretrained
         tokenizer_class = self.__class__.__name__
+        # import here to prevent circular import error
+        from .tokenization_utils_fast import PreTrainedTokenizerFast
```

Comment:
would love to avoid this, it's pretty weird

Reply:
> would love to avoid this, it's pretty weird

Another option is to remove this block of `Fast` strip code completedly. Is this legacy code still needed now?

Reply:
We are not using different, clean, method to perform the same fix.